patches/minecraft/net/minecraft/client/renderer/FogRenderer.java.patch

```diff
@@ -36,6 +29,7 @@
        FogShape fogshape = FogShape.SPHERE;
        float f;
        float f1;
++      // TODO: remove this hook in 1.19
 +      float hook = net.minecraftforge.client.ForgeHooksClient.getFogDensity(p_109026_, p_109025_, partialTicks, 0.1F);
```

Comment:
The hook method should be terminally deprecated (`@Deprecated(forRemoval = true, since = "1.18.2")`, `/** @deprecated To be removed in 1.19, because ... */`).